ramls/loan-storage.raml

```diff
@@ -28,6 +29,27 @@ resourceTypes:
   get-only: !include raml-util/rtypes/get-only.raml
 
 /loan-storage:
+  /anonymizeloans:
+    post:
+      is: [validate]
+      body:
+        application/json:
+          type: string[]
+      responses:
+        200:
+          description: "Number of closed loans for this user that have been anonymized"
+          body:
+            application/json:
+              type: anonymize-loans-response
+        422:
```

Comment:
422 response is redundat here as it is already included in `validate` (see validation.raml)

Reply:
Done.